src/containers/project_issue.py

```diff
@@ -1,18 +1,82 @@
-from dataclasses import dataclass, asdict
 from typing import Optional
 
+NO_VALUE_SELECTED = "N/A"
+
 
-@dataclass
 class ProjectIssue:
-    Number: int
-    Owner: str
-    RepositoryName: str
-    Title: str
-    State: str
-    Status: Optional[str] = None
-    Size: Optional[str] = None
-    Priority: Optional[str] = None
-    MSCW: Optional[str] = None
+    def __init__(self):
+        self.number: int = 0
+        self.organization_name: str = ""
+        self.repository_name: str = ""
+        # TODO: title and state can be deleted, since they are already mined in repository Issue
```

Comment:
Look at GithubManager proposal and issue and repository classes.

Reply:
implemented. Comment can be closed.